.eslintrc.cjs

```diff
@@ -15,7 +15,7 @@ module.exports = {
 		'eqeqeq': ['error', 'always', { 'null': 'ignore' }],
```

Comment:
This file is the only remnant of CommonJS in the repo (apart from the dependencies which I haven't updated yet).

In the upcoming ESLint v9, the new flat config format will also be in Node ESM format, and we'll have to move ESLint config to ESM at that point.